### PR TITLE
Prometheus Adapter Module

### PR DESCRIPTION
**Pull request by Bslabe123**
Module gives us the ability to set the averageValue in a HPA object to the value of a promql query unlike custom metrics stackdriver adapter.

### CODE REVIEW

**Comment by liurupeng** [comment]
could you provide an example of using HBM utilization percentage for HPA with promql on JetStream?

**Comment by Bslabe123** [comment]
> HBM utilization percentage for HPA with prom

This is configurable in the Jetstream module which is currently in a [separate PR](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/719/files#diff-0d969817cd1020344adfef8e2612a5118546274bb9e2a446c14eb762de0e8921)

File: modules/prometheus-adapter/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+   - (optional) PROMETHEUS_HELM_VALUES_FILE: Values file to pass when deploying `prometheus-community/prometheus-adapter` chart
+
+```
+curl https://raw.githubusercontent.com/GoogleCloudPlatform/prometheus-engine/v0.10.0/examples/frontend.yaml | envsubst
+
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+
+if [ -z "$PROMETHEUS_HELM_VALUES_FILE" ]
+    helm install jetstream prometheus-community/prometheus-adapter
```
Comment (annapendleton) [inline modules/prometheus-adapter/README.md:16]: Does this work for non jetstream workloads?

If so, we should parameterize the name "jetstream" here

**Review by annapendleton — APPROVED** [review]

**Pull request merged**